src/main/groovy/io/micronaut/build/MicronautBuildCommonPlugin.groovy

```diff
@@ -45,7 +45,7 @@ class MicronautBuildCommonPlugin implements Plugin<Project> {
         }
         def byteBuddyVersionProvider = versionProviderOrDefault(project, 'bytebuddy', '1.12.18')
         def objenesisVersionProvider = versionProviderOrDefault(project, 'objenesis', '3.1')
-        def logbackVersionProvider = versionProviderOrDefault(project, 'logback', '1.2.3')
+        def logbackVersionProvider = versionProviderOrDefault(project, 'logback', List.of("mnLogging", "libs"), '')
```

Comment:
Should we change the order of catalogs so we can override logback version from 'mnLogging' catalog with version from 'libs' catalog if necessary?
Also, what if a project doesn't have logback in 'libs' and doesn't import micronaut-logging, will the build fail with empty default version?

Reply:
The default should be `1.4.7` not `1.2.3` in master.

Reply:
I don't think that the build plugins should provide a default version. That's why I removed it altogether. The default version should be read from properties file or the catalogs, so indeed, builds will fail if it's not declared, which is a breaking change.

As for inverting the order, I'm not sure it makes sense.

Reply:
I have pushed a change to prioritize `libs` over `mnLogging`. In addition, the logback dependency will only be added to `testImplementation` if the logback version is defined (either explicitly or via a catalog).